### DISCUSSION TITLE
recipes/holiday/monastery-pumpkin-bread/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#877] [Comments]
# recipes/holiday/monastery-pumpkin-bread/

A collection of my recipes.

https://nicholaswilde.io/recipes/holiday/monastery-pumpkin-bread/

**Comment by nicholaswilde** [comment]
1973 LA Times
2 cups pumpkin ~1 15-oz can
Grease tins with veg shortening
Cool in pan ~5 min, then remove and reinsert into pan. Use knife to run around edge.

**Reply by nicholaswilde** [reply]
https://www.latimes.com/food/dailydish/la-dd-monastery-pumpkin-bread-20130930-story.html

**Reply by nicholaswilde** [reply]
Tag: favorite.
